Lib/fontTools/otlLib/builder.py

```diff
@@ -2574,7 +2574,8 @@ def add(self, glyphs):
             return
         self.classes_.add(glyphs)
         for glyph in glyphs:
-            assert glyph not in self.glyphs_
+            if glyph in self.glyphs_:
+                raise TypeError(f"Glyph {glyph} already present in class.")
```

Comment:
it looks more like a `ValueError` than a `TypeError` to me

Reply:
Or ideally this would raise a custom exception class. See if otlLib already has one, if not try make one

Reply:
🧠〰🧠

Reply:
I had this is a KeyError originally and did an amend commit to change them all to ValueErrors, obviously I made a mistake! I'll make a custom exception.

Reply:
Decided to go ahead with a ValueError.